bot/exts/easter/earth_photos.py

```diff
@@ -0,0 +1,48 @@
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+from bot.constants import Tokens
+
+UnClient_id = Tokens.unsplash_key
```

Comment:
Could you please remove this wrapper and use `Tokens.unsplash_key` directly in your param's dict.

Reply:
I've done this, i just dont have it commited yet as far as I know